_events/2024-11-30-functionalscala.md

```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: Functional Scala 2024
+logo: /resources/img/functional-scala-logo.png
+location: London, UK
+description: "Functional Scala 2024 is a two-day conference in London with amazing speakers from around the world passionate about Scala."
+start: 30 November 2024
+end: 1 December 2024
+link-out: https://www.eventbrite.com/e/functional-scala-2024-tickets-771636073267
```

Comment:
it's not at https://www.functionalscala.com/ yet, will it be soon? Maybe that's a better link.

Reply:
I think that doesn't matter, and the site will finally be updated.

Reply:
Not sure! Anyway, whenever the site is updated, we can update this link to refer to the main site.